chainerio/__init__.py

```diff
@@ -146,6 +310,12 @@ def remove(path: str, recursive: bool = False) -> None:
 
 
 def get_root_dir() -> str:
```

Comment:
I checked the implementation at `_context.py` and it seems not working properly as the table.